.github/workflows/kind-e2e.yaml

```
@@ -22,6 +22,8 @@ jobs:
         - v1.21.1/kourier
         - v1.22.2/istio
         - v1.22.2/gateway-api
+        - v1.23.0/istio
+        - v1.23.0/gateway-api
```

I don't think we need both. We only had both on 1.22.2 because there's 4 net-* plugins but only 3 K8s versions.

As such, I'd suggest

```suggestion
        - v1.23.0/gateway-api
```

---

Now the k8s min version (v1.20) was bumped to v1.21 as https://github.com/knative/pkg/pull/2397 so we need to bump the list like:

```
        - v1.21.1/contour
        - v1.22.2/kourier
        - v1.23.0/istio
        - v1.23.0/gateway-api
```